src/dotnet-archive/Program.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.DotNet.Cli.CommandLine;
+//using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Archive;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.DotNet.Tools.Archive
+{
+
+    public partial class ArchiveCommand
+    {
+        public static int Main(string[] args)
+        {
+            //DebugHelper.HandleDebugSwitch(ref args);
+
+            var app = new CommandLineApplication();
+            app.Name = "archive";
+            app.FullName = ".NET archiver";
+            app.Description = "Archives and expands sets of files";
+            app.HelpOption("-h|--help");
+
+            var extract = app.Option("-x|--extract <outputDirectory>", "Directory to extract to", CommandOptionType.SingleValue);
```

Comment:
I think it would be worth mentioning in the help here that `-x` and `<sources>` are mutually exclusive.